File: Euclid.Storage/Database/Data/Misc/SettingsData.cs
using FluentNHibernate.Mapping;

namespace Euclid.Storage.Database.Data
{
    public class SettingsDataMapping : ClassMap<SettingsData>
    {
        public SettingsDataMapping()
        {
            Table("server_settings");
            Id(x => x.Key, "setting");
            Map(x => x.Value, "value");
        }
    }

    public class SettingsData
    {
        public virtual string Key { get; set; }
        public virtual string Value { get; set; }
    }
}

File: euclid-storage/src/main/java/euclid/storage/database/data/SettingsData.java
package euclid.storage.database.data;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "server_settings")
public class SettingsData {

    @Id
    @Column(name = "setting")
    private String key;

    @Column(name = "value")
    private String value;

    public SettingsData() {
    }

    public SettingsData(String key, String value) {
        this.key = key;
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}
